src/api.rs

```diff
@@ -97,6 +97,22 @@ pub struct BlockSummary {
     pub merkle_root: bitcoin::hash_types::TxMerkleNode,
 }
 
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+pub struct AddressStats {
+    pub address: String,
+    pub chain_stats: AddressTxnSummary,
+    pub mempool_stats: AddressTxnSummary,
+}
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Deserialize)]
```

Comment:
I like to derive `Copy` whenever possible especially for libraries, because we don't know how the end user is going to use it. It would be nicer for them to be able to Copy and dereference it instead of cloning it.

Reply:
Can we merge this in and I can go through the API and add Copy to the other structs that make sense? @oleonardolima ?

Reply:
I'm fine with it, I'd like to see what others think about it.